api/cluster-resources/aou-upload-policy-extension.js

```diff
@@ -0,0 +1,22 @@
+// Workbench "Upload policy" extension, to show the policy before letting user upload any file
+
+define([
+  'base/js/namespace'
+], (Jupyter) => {
+
+  const load_upload_extension = () => {
+  // This will open a dialog box when the user clicks Upload.
+  // File will be uploaded only if the user clicks OK else it will just close the dialog box and do nothing.
+
+  $("#notebook_list_info input").click(function() {
+    return confirm( "It is All of Us data use policy to not upload data or files containing " +
```

Comment:
Italicize `All of Us` if possible. Not sure whether HTML is supported here

Reply:
I did try that however HTML isnt supported in confirm box its a plain string. Should i replace this with customized one?